doc/source/release-notes/Percona-Server-5.5.51-38.1.rst

```diff
@@ -0,0 +1,62 @@
+.. rn:: 5.5.51-38.1
+
+==============================
+ |Percona Server| 5.5.51-38.1
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.51-38.1 on
+August 19th, 2016. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.51-38.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.51
+<http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-51.html>`_, including
+all the bug fixes in it, |Percona Server| 5.5.51-38.1 is now the current stable
+release in the 5.5 series. All of |Percona|'s software is open-source and free,
+all the details of the release can be found in the `5.5.51-38.1 milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.5.51-38.1>`_.
+
+Bugs Fixed
+==========
+
+ :ref:`pam_plugin` would abort authentication if there were more than a
+ thousand members in the group. Bug fixed :bug:`1608902`.
+
+ If ``DROP DATABASE`` would fails to delete any of the tables, command is still
+ added to the binary log as ``DROP TABLE t1, t2, ...`` for the tables for which
+ drop succeeded. This could break the replication if a slave has a different
+ schema so that it references any of the dropped tables. Bug fixed
+ :bug:`1525407` (upstream :mysqlbug:`79610`).
+
+ :ref:`pam_plugin` didn't support spaces in the group names. Bug fixed
+ :bug:`1544443`.
+
+ |Percona Server| 5.5 could not be built with the
+ ``-DMYSQL_MAINTAINER_MODE=ON`` option. Bug fixed :bug:`1590454`.
+
+ If the mysql client (or ``libmysqlclient``) was reading from a connected
```

Comment:
In the client library, any EINTR received during network I/O was not handled correctly